pennylane/qnn/__init__.py

```diff
@@ -11,9 +11,49 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""This module contains classes and functions for constructing quantum neural networks from QNodes."""
+"""
+This module contains functionality for converting PennyLane QNodes into layers that are compatible
+with Keras and PyTorch.
 
-import pennylane.qnn.cost
+.. note::
 
-from .keras import KerasLayer
-from .torch import TorchLayer
+    Check out our :doc:`Keras <demos/tutorial_qnn_module_tf>` and
+    :doc:`Torch <demos/tutorial_qnn_module_torch>` tutorials for further details.
+
+
+.. rubric:: Classes
+
+.. autosummary::
+    :toctree: api
+    :nosignatures:
+    :template: autosummary/class_no_inherited.rst
+
+    ~KerasLayer
+    ~TorchLayer
+
+
+.. rubric:: Modules
+
+.. autosummary::
+    :toctree: api
+    :nosignatures:
+
+    ~cost
```

Comment:
```suggestion
```
Optional, but we don't currently include this module in the docs.

Reply:
Ah, I thought this was a bug that it wasn't included?